src/lightning_lite/plugins/collectives/single_device_collective.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, List, Optional
+
+import torch
+
+from lightning_lite.plugins.collectives.collective import Collective
+from lightning_lite.utilities.types import CollectibleGroup
+
+
+class SingleDeviceCollective(Collective):
+    @property
+    def rank(self) -> int:
+        return 0
+
+    @property
+    def world_size(self) -> int:
+        return 1
+
+    def broadcast(self, tensor: torch.Tensor, *_: Any, **__: Any) -> torch.Tensor:
```

Comment:
should we name them args and kwargs like usual or do we say since they should not be used we don't care?

Reply:
that is exactly the reason, why they are `_` and `__` -> to indicate, they are very much not used